### DISCUSSION TITLE
Equipo 16 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#23] [Proyecto Final]
Hola Equipo 16, (¡Mary Ann!)
Aquí está el hilo para actualizarnos sobre sus avances con el proyecto final, charlar con el Equipo de Educación, recibir retroalimentación sobre cada etapa, etcetera.
Saludos,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @MaryAnnHernani
Espero estés muy bien. Como discutimos en la asesoría, tienes prórroga para la etapa 0 del proyecto final hasta este domingo. La etapa se trata de elegir un tema para tu proyecto y redactar una lista de 3-5 preguntas para la entrevista. 
Para entregar la etapa 0, solo tienes que rellenar este formulario: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-16/etapa-0/README.md
Cualquier consulta, aquí ando.
Kasturi

**Comment by Debora-Cuestas** [comment]
Hola @MaryAnnHernani, cómo estás?
Mi nombre es Debora y soy moderadora de PH, mucho gusto!. Estoy ayudando a Kasturi a brindar retroalimentación de las etapas que has entregado! 
Ante todo, me gustaría felicitarte por el tema que elegiste y también, me gustaría preguntarte ¿Por qué has escogido este tema y de qué forma afecta esta problemática en tu comunidad? (Sólo si gustas responder, estoy muy interesada en esta experiencia.)

_En cuanto a la_ **Etapa 0**:
-Me gustan tus preguntas para conocer al sistema actual de enseñanza y conocer cómo se puede mejorar el sistema actual o simplemente saber cómo podría captarse mejor la información (entiendo que esto puede depender de muchos factores).

Felicitaciones! <3

_En cuanto a la_ **Etapa 1**:
-En primer lugar felicitarte por el mapa!
-En Segundo lugar felicitarte porque has logrado 3 entrevistas con docentes. Creo que esto nutre el proyecto ya que, tienes distintas perspectivas que pueden ayudar a solventar esta problemática o mismo encontrar un punto en común.

Mi sugerencia sería:
-Hacer un mapa para cada uno de los entrevistados para analizar perspectivas y conectar ideas. Así también podrás comparar en qué coinciden los entrevistados y qué decisiones implementadas han resultado más potables y cuales no tanto. 
Recuerda que cada persona es un mundo :)

_En cuanto a la_ **Etapa 2**:

Felicitaciones por llegar hasta aquí y gracias por grabar el vídeo y enseñarnos tu trabajo!
Entiendo que has elegido el 8S como método de lluvia de ideas y adoré las ideas (valga la redundancia) que has añadido y cómo has optimizado todo con el método SCAMPER. Me gusta mucho que hayas traído a colación el método Singapur! 

Por otro lado, te comento que al parecer no se han cargado las imágenes de tu lluvia de ideas en aquí en tu carpeta de Github, si pudieses ajustarlo sería genial! Sólo si es posible, de cualquier manera se aprecian de forma clara en el vídeo. :)

Te felicito y quedo al pendiente de la entrega de la Etapa 03. Sigue así, mucha suerte!

Cualquier duda aquí me encuentro 👍🏻 :D

**Comment by Debora-Cuestas** [comment]
Hola @MaryAnnHernani, cómo estás? Espero que bien.
He visto (un poco tarde,  me disculpo) que aún no has podido entregar la etapa 3 proyecto, estamos dispuestsa a dar prorroga hasta la fecha limite que quieras. Tampoco es necesario entregar etapas pulidas, lo importante es practicar y aplicar los conceptos al proyecto (palabras textuales de @KasturiSShah. Quedamos al pendiente.

Cualquier inquietud o inconvenientes puedes escribirnos! :)

**Comment by MaryAnnHernani** [comment]
Buenos días, mil disculpa Debora el día de hoy me pondré al día por favor
si pudieran darme hasta el domingo para terminar les agrdecería mucho.

**Reply by KasturiSShah** [reply]
Hola @MaryAnnHernani buena tarde,
No te preocupes. Claro que sí, tendrás hasta este domingo para ponerte al corriente. Avísanos  si pudiéramos  apoyarte en algo.
Kasturi

**Comment by KasturiSShah** [comment]
Hola @MaryAnnHernani 

Espero estés bien y avances con las etapas 3, 4 y 5. Te escribo con algunos recordatorios y sugerencias sobre cómo terminar el proyecto de manera eficiente.

¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar su presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si les conviene la asignación de su equipo a una de las dos sesiones de clausura del curso, o si quieren cambiar su día asignado al otro día. Horario disponible [aquí](https://docs.google.com/spreadsheets/d/1GDWqDjb5OVqz2vWRM_DoO0wrFM4o14gEICREDq8ZxQ4/edit?usp=sharing))

**_No es necesario tener un proyecto pulido ni completado para presentarlo. Sería excelente en la clausura del curso simplemente mostrar el trabajo que has hecho y describir los aspectos que habrías desarrollado más si hubieras tenido más tiempo._**

Mi recomendación sería poner 1-2 días de trabajo en completar la etapa 3, 4 y 5, y luego preparar para la presentación final. La etapa 5 no necesita que la etapa 3 ni 4 sea completada. Entonces, puedes pasar 30 minutos completando la etapa 5, entregarla, y luego avanzar con las etapas 3 y 4.

Cualquier pregunta sobre el proyecto final, déjanos saber. Quedamos atentas a tus actualizaciones para poder revisar las etapas / apoyarte en lo que podamos.

Mucha suerte, mucho éxito, quedo atenta cualquier cosa
Kasturi

**Comment by MaryAnnHernani** [comment]
Gracias 🙏 mañana y el lunes terminaré las 3 etapas Enviado desde mi iPhoneEl 22 abr. 2023, a la(s) 09:45, KasturiSShah ***@***.***> escribió:﻿
Hola @MaryAnnHernani
Espero estés bien y avances con las etapas 3, 4 y 5. Te escribo con algunos recordatorios y sugerencias sobre cómo terminar el proyecto de manera eficiente.
¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar su presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si les conviene la asignación de su equipo a una de las dos sesiones de clausura del curso, o si quieren cambiar su día asignado al otro día. Horario disponible aquí)
No es necesario tener un proyecto pulido ni completado para presentarlo. Sería excelente en la clausura del curso simplemente mostrar el trabajo que has hecho y describir los aspectos que habrías desarrollado más si hubieras tenido más tiempo.
Mi recomendación sería poner 1-2 días de trabajo en completar la etapa 3, 4 y 5, y luego preparar para la presentación final. La etapa 5 no necesita que la etapa 3 ni 4 sea completada. Entonces, puedes pasar 30 minutos completando la etapa 5, entregarla, y luego avanzar con las etapas 3 y 4.
Mucha suerte, mucho éxito, quedo atenta cualquier cosa
Kasturi

**Comment by Debora-Cuestas** [comment]
Buen día @MaryAnnHernani, cómo estás?
Noté que aún no pudiste entregar las etapas, necesitas ayuda? No te preocupes si aun no pudiste terminarlo. Mañana estaremos en el evento de clausura y me gustaría saber si estarás ahí?

Espero que estés bien, cualquier duda quedamos al pendiente con @KasturiSShah :)

**Comment by MaryAnnHernani** [comment]
SI.

**Reply by Debora-Cuestas** [reply]
Excelente @MaryAnnHernani! Gracias por confirmar! 
Espero puedas completar/adelantar tus etapas restantes, cualquier duda nos avisas. Nos vemos mañana :D

**Comment by MaryAnnHernani** [comment]
Kasturi buenos días 😃 disculpa no podré estar en la presentación de hoy tuve una emergencia crees que pueda presentarte el trabajo el lunes. Gracias 🙏 por tu ayuda Enviado desde mi iPhoneEl 22 abr. 2023, a la(s) 10:22, Mary Ann ***@***.***> escribió:﻿Gracias 🙏 mañana y el lunes terminaré las 3 etapas Enviado desde mi iPhoneEl 22 abr. 2023, a la(s) 09:45, KasturiSShah ***@***.***> escribió:﻿
Hola @MaryAnnHernani
Espero estés bien y avances con las etapas 3, 4 y 5. Te escribo con algunos recordatorios y sugerencias sobre cómo terminar el proyecto de manera eficiente.
¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar su presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si les conviene la asignación de su equipo a una de las dos sesiones de clausura del curso, o si quieren cambiar su día asignado al otro día. Horario disponible aquí)
No es necesario tener un proyecto pulido ni completado para presentarlo. Sería excelente en la clausura del curso simplemente mostrar el trabajo que has hecho y describir los aspectos que habrías desarrollado más si hubieras tenido más tiempo.
Mi recomendación sería poner 1-2 días de trabajo en completar la etapa 3, 4 y 5, y luego preparar para la presentación final. La etapa 5 no necesita que la etapa 3 ni 4 sea completada. Entonces, puedes pasar 30 minutos completando la etapa 5, entregarla, y luego avanzar con las etapas 3 y 4.
Mucha suerte, mucho éxito, quedo atenta cualquier cosa
Kasturi

**Reply by Debora-Cuestas** [reply]
Buen dia @MaryAnnHernani, cómo estás?
No creo que @KasturiSShah tenga problema.
Es una de pena el no poder despedirnos hoy 😿 espero que estés bien y aquí nos vemos cualquier cosa que necesites 🙌

**Reply by KasturiSShah** [reply]
Hola @MaryAnnHernani 
Espero todo bien. No te preocupes. ¿Qué tal el miércoles a las 17:40 CDMX para tu presentación? @Debora-Cuestas y yo estaremos presentes 😄 
Saludos
Kasturi

**Comment by MaryAnnHernani** [comment]
Perfecto, mcuhas gracias ahí estaré

**Reply by Debora-Cuestas** [reply]
Excelente! Buen fin de semana y felíz día del trabajador 🙌

**Reply by Debora-Cuestas** [reply]
Hola @MaryAnnHernani, nos vemos en unos minutos ? 🤗

**Comment by Debora-Cuestas** [comment]
Hola @MaryAnnHernani , cómo estás?
Disculpa la hora, ante todo quería agradecerte por todo el trabajo que has hecho y felicitarte por este esfuerzo tan grande que estás llevando a cabo, gracias por querer llevar el conocimiento tan lejos y a lugares que parecen recónditos pero tesoros como tu lo han encontrado para pulirlos. Simplemente, muchas gracias por tu esfuerzo. Aqui te dejo la pagina de Udemy que cada tanto sacan cursos gratis por tiempo limitado, estate al pendiente para no perderlos! -para diversas categorias-

https://www.udemy.com/es/topic/programming-fundamentals/?lang=es&price=price-free&sort=popularity


Cualquier duda estamos al pendiente! :)

**Comment by MaryAnnHernani** [comment]
Muchísimas gracias querida Debora espero volver a contactarnos pronto.
